package.json

```diff
@@ -16,6 +16,7 @@
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "jest -c ./test/unit/jest.config.ts",
     "test:e2e": "playwright test -c ./test/e2e/playwright.config.ts",
+    "pretest:e2e": "rm -rf ./test/e2e/chromeData/*",
     "posttest:e2e": "rm -rf ./test/e2e/chromeData/*",
```

Comment:
Why do we need to clean before and after?

Reply:
Added because when a test fails playwright starts a server for the results that needs an interrupt signal to stop. That interrupt signal prevents the post command from cleaning up.

But on second thought, I'll just create an explicit script to cleanup that runs regardless of the main scripts completion.

Reply:
Turns out, you can't run a script after a SIGINT because of how npm handles SIGINT. So I'm leaving in the pre script.